docs/source/item-validation.rst

```diff
@@ -157,9 +157,13 @@ as a `dict`:
 
     # settings.py
 
-    SPIDERMON_VALIDATION_MODELS: {
-        DummyItem: 'myproject.spiders.validators.DummyItemModel',
-        OtherItem: 'myproject.spiders.validators.OtherItemModel',
+    from tutorial.items import DummyItem, OtherItem
+
+    ...
```

Comment:
Maybe `# …`?

Reply:
hello @ Gallaecio I just fixed this conflict,